db/migrations/20201020204056_index_kpack_lifecycle_data_droplet_guid.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  up do
+    alter_table :kpack_lifecycle_data do
+      add_index :droplet_guid, name: :kpack_lifecycle_droplet_guid_index
```

Comment:
This strikes me as an obvious improvement, but I've never thought about indices on `lifecycle_data` tables prior to now. It seems likely that we've missed some other indices on this table in the past `lifecycle_data` links itself to all sorts of other objects...

Reply:
I put a little blurb in the commit message about this one:

```
- add index to kpack_lifecycle_data.droplet_guid
  - this is important since droplets find their associated lifecycle
  through this column
  - a similar index exists on buildpack_lifecycle_data
```

I think it's important if we want the `droplet.kpack?` type of methods to be performant.